src/context.rs

```diff
@@ -1709,21 +1677,17 @@ impl BlockContext {
 
     match (has_above, has_left) {
       (true, true) => {
-        let above_intra = !self.above_of(bo).is_inter();
-        let left_intra = !self.left_of(bo).is_inter();
+        let above_intra = !self.blocks[bo.y - 1][bo.x].is_inter();
```

Comment:
Well, to me this comes like 'rephrasing' and make my debugging much more quicker.

Reply:
Out of curiosity, why `self.blocks[bo.y - 1][bo.x]` makes your debugging easier than `self.above_of(bo)`?

Reply:
Well, quite exaggerated, but because I don't need to the function and check the self.blocks[bo.y][bo.x] and its neighbors including the one being read [bo.y-1][bo.x], that is one step deeper. Maybe would not make good sense to others :)